rubicon/objc/objc.py

```diff
@@ -1314,6 +1326,65 @@ def __setattr__(self, name, value):
                 super(ObjCInstance, type(self)).__setattr__(self, name, value)
 
 
+class ObjCListInstance(ObjCInstance):
+    def __getitem__(self, item):
+        if item >= self.count:
+            raise IndexError('list index out of range')
+        return self.objectAtIndex(item)
+
+    def __setitem__(self, item, value):
+        try:
+            self.insertObject_atIndex_(value, item)
```

Comment:
I'm not sure, but isn't [`-[NSMutableArray insertObject:atIndex:]`](https://developer.apple.com/reference/foundation/nsmutablearray/1416682-insertobject?language=objc) for *inserting* an extra item into the array (like `list.insert`)? [`-[NSMutableArray replaceObjectAtIndex:withObject:]`](https://developer.apple.com/reference/foundation/nsmutablearray/1414510-replaceobjectatindex?language=objc) looks like the correct method for reassigning an object at a given index.